docs/src/pages/system/basics/basics.md

```diff
@@ -1,338 +1,215 @@
-# @material-ui/system
+# Material-UI System
 
-<p class="description">Styled system & style functions for building powerful design systems.</p>
+<p class="description">Utility-first style functions for rapidly building custom design systems.</p>
 
-## Getting Started
+The system lets you quickly build custom UI components leveraging the design tokens defined in your theme.
```

Comment:
The very first page requires knowledge of "design token". Either we explain it here or link to pre-requisites to understand this page. Otherwise we'll have cliff in the very first sentence.

Reply:
```suggestion
The system lets you quickly build custom UI components leveraging the design tokens defined in your theme. You can think of the `design tokens` as any theme values that can be used across different parts of your application.
```

Do you think this would be enough?

Reply:
Why not just say "theme values" then?